src/main/java/network/brightspots/rcv/GuiConfigController.java

```diff
@@ -78,6 +80,47 @@
       DateTimeFormatter.ofPattern("yyyy-MM-dd");
   private static final String CONFIG_FILE_DOCUMENTATION_FILENAME =
       "network/brightspots/rcv/config_file_documentation.txt";
+  private static final List WINNER_ELECTION_MODE_VALUES = List.of(
+      "Single-winner majority determines winner",
+      "Multi-winner allow only one winner per round",
+      "Multi-winner allow multiple winners per round",
+      "Bottoms-up",
+      "Bottoms-up using percentage threshold",
+      "Multi-pass IRV"
+  );
```

Comment:
It's unfortunate that we're defining these strings here and then repeating them for the map definition. Can't we just use the map definition and then pull these strings out when we need them?

Same comment for tiebreak modes.

Reply:
Totally agree. I tried with just the map at first, but the keys were unsorted so they appeared in a wacky order. Then I tried a sorted map, but there's apparently still no immutable version in vanilla Java. This was the somewhat unhappy compromise I made without needing to bring another library like Guava in. I do think it'll be improved a little by factoring the string literals out at least. The things I do for defensive programming...